doc/usage/al2023.md

```diff
@@ -5,6 +5,7 @@
 <!-- template-variable-table-boundary -->
 | Variable | Description |
 | - | - |
+| `accelerator_vendor` | Vendor that provides the GPU or accelerator hardware. Currently we support Neuron and NVIDIA. |
```

Comment:
not loving this name 😂 maybe `enable_accelerator`? what else have you considered?

also should mention in the doc for `nvidia_driver_major_version` that it's only used when this var == `nvidia`

Reply:
>what else have you considered?

you don't wanna know :)

I'm good with `enable_accelerator`